wiki/en/en-Setup.md

```diff
@@ -0,0 +1,74 @@
+---
+layout: wiki
+title: "Setup"
+lang: "en"
+permalink: "/wiki/Setup"
+---
+
+# Setup
+
+To get the best from Jamulus, at a minimum you will need:
+
+1. **A wired internet connection** (not wi-fi - see the [FAQ](/wiki/FAQ))
+1. **Wired headphones** (not Bluetooth or speakers - see the [FAQ](/wiki/FAQ))
+1. **A reasonable audio device, sound card and/or mic** ([see this list](/kb/2021/01/05/Jamulus-Sound-Devices.html) for examples)
+
+If you have any questions, please see the [Discussions](https://github.com/jamulussoftware/jamulus/discussions)
+
+## Installation
+To get Jamulus working with your hardware, please follow the installation guide for your platform:
+
+* [Windows](/wiki/Installation-for-Windows)
+* [macOS](/wiki/Installation-for-Macintosh)
+* [Linux](/wiki/Installation-for-Liknux)
+
+And for now, close all other applications. It’s best to start simple at first.
+
+### Jamulus main window
+
+When you open Jamulus, you will see a window which looks like this:
+
+[screenshot]
+
+
+#### Setting up a profile
+
+First, let others know who you are. Click on “View” (bar on the top) > “My Profile…” Now you will see this:
+
+[screenshot]
+
+
+Fill in at least “Alias/Name” and close the window.
+
+### Connecting to a server and testing your sound
+
+**Before playing with others, we recommend that you connect to an empty server to test your setup**, and make sure you are listening to the signal coming back from the server (if possible) and not yourself.
+
+Before you connect to a server, you should **not** be able to hear yourself.  Listening to your sound from Jamulus (and not from yourself) is called “The Golden Rule” and enables you to play in time with others ([see the FAQ](/wiki/FAQ)). 
+
+Now use the “Connect” button in the Jamulus main window to join a server. A window will now open:
+
+[screenshot]
+
+The most important thing about servers is their “ping time”. The bigger the number, the harder it will be to play in time with others. Usually, you would select a server with a ping of less than 15ms if you can. 
+
+Once connected to a server, make sure you can hear yourself OK and fix any input volume or other problems. And of course, check that you are also able to obey [The Golden Rule](/wiki/Client-Troubleshooting).
+
+## Playing for the first time
+
+With your sound all set up, you are ready to go. When you connect to a server (you may want to select a genre from the list) and the sliders you see on the right are your own personal mix. Everything you change here will change what you hear, but won’t affect others. If you move a slider down, that user will be quieter, if you move it up, that user will be louder for you.
+
+[screenshot]
+
+If you don’t want others to hear your audio, click on the “Mute Myself” button which will stop your audio from being sent to other musicians. They won’t be able to tell you have done this though. And if you see a “mute” icon next to a player, that means they can’t hear you. 
```

Comment:
> And if you see a “mute” icon next to a player, that means they can’t hear you. 

This seems a bit out of context.  Maybe too advanced for now?

Reply:
This was out of an attempt to explain muting as it relates to "mute myself" in general. So maybe it should be "But if you see a “mute” icon next to a player..."

Reply:
Ah, suddenly... Yes, "But..." makes it clear!  I'd add "because they've chosen to mute your mixer channel in their mix" (or something simpler but along those lines).  Otherwise I think it's still a bit disorienting.